wicket-jquery-ui-plugins/src/main/resources/com/googlecode/wicket/jquery/ui/plugins/wysiwyg/resource/js/bootstrap-wysiwyg.min.js

```diff
@@ -6,5 +6,4 @@
"wysiwyg-html-mode")!==true){var oContent=editor.html();var editorPre=$("<pre />");$(editorPre).append(document.createTextNode(oContent));$(editorPre).attr("contenteditable",true);$(editor).html(" ");$(editor).append($(editorPre));$(editor).attr("contenteditable",false);$(editor).data("wysiwyg-html-mode",true);$(editorPre).focus()}else{$(editor).html($(editor).text());$(editor).attr("contenteditable",true);$(editor).data("wysiwyg-html-mode",false);$(editor).focus()}};Wysiwyg.prototype.insertFiles=function(files,options,editor,toolbarBtnSelector){var self=this;editor.focus();$.each(files,function(idx,fileInfo){if(/^image\//.test(fileInfo.type)){$.when(self.readFileIntoDataUrl(fileInfo)).done(function(dataUrl){self.execCommand("insertimage",dataUrl,editor,options,toolbarBtnSelector);editor.trigger("image-inserted")}).fail(function(e){options.fileUploadError("file-reader",e)})}else{options.fileUploadError("unsupported-file-type",fileInfo.type)}})};Wysiwyg.prototype.markSelection=function(input,color,options){this.restoreSelection();if(document.queryCommandSupported("hiliteColor")){document.execCommand("hiliteColor",false,color||"transparent")}this.saveSelection();input.data(options.selectionMarker,color)};Wysiwyg.prototype.bindToolbar=function(editor,toolbar,options,toolbarBtnSelector){var self=this;toolbar.find(toolbarBtnSelector).click(function(){self.restoreSelection();editor.focus();if(editor.data(options.commandRole)==="html"){self.toggleHtmlEdit(editor)}else{self.execCommand($(this).data(options.commandRole),null,editor,options,toolbarBtnSelector)}self.saveSelection()});toolbar.find("[data-toggle=dropdown]").click(this.restoreSelection());toolbar.find("input[type=text][data-"+options.commandRole+"]").on("webkitspeechchange change",function(){var newValue=this.value;this.value="";self.restoreSelection();if(newValue){editor.focus();self.execCommand($(this).data(options.commandRole),newValue,editor,options,toolbarBtnSelector)}self.saveSelection()}).on("focus",function(){var input=$(this);if(!input.data(options.selectionMarker)){self.markSelection(input,options.selectionColor,options);input.focus()}}).on("blur",function(){var input=$(this);if(input.data(options.selectionMarker)){self.markSelection(input,false,options)}});toolbar.find("input[type=file][data-"+options.commandRole+"]").change(function(){self.restoreSelection();if(this.type==="file"&&this.files&&this.files.length>0){self.insertFiles(this.files,options,editor,toolbarBtnSelector)}self.saveSelection();this.value=""})};Wysiwyg.prototype.initFileDrops=function(editor,options,toolbarBtnSelector){var self=this;editor.on("dragenter dragover",false).on("drop",function(e){var dataTransfer=e.originalEvent.dataTransfer;e.stopPropagation();e.preventDefault();if(dataTransfer&&dataTransfer.files&&dataTransfer.files.length>0){self.insertFiles(dataTransfer.files,options,editor,toolbarBtnSelector)}})};$.fn.wysiwyg=function(userOptions){var wysiwyg=new Wysiwyg(this,userOptions)}})(window,window.jQuery);
```

Comment:
Did you minimize it manually ? :-) (s/\n//g ?!)

Reply:
@martin-g I run `uglify` from command line and add header manually :(
Will create pr to original repo ...

Reply:
PR created https://github.com/steveathon/bootstrap-wysiwyg/pull/153